helm-app-operator/Gopkg.toml

```diff
@@ -65,9 +77,8 @@ required = [
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
-  branch = "master"
-  # version = "=v0.0.6"
+  version = "=v0.3.0"
```

Comment:
Took so long to review this that we now have `v0.4.0` in case you want to update to that.